docs/installing.rst

```diff
@@ -77,3 +77,23 @@ Make sure that your PATH includes the standard
 location for Python scripts (e.g. ~/anaconda/bin)
 
 
+Building the public Data Bases
```

Comment:
It is not intended that the typical user will contribute to building these databases.  Really, all they will want is to download them.

And that is what you describe.  So let's replace "Build" with "Download" or "Getting"